src/main/java/Lesson4/Task1/Main.java

```diff
@@ -0,0 +1,26 @@
+package Lesson4.Task1;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        AbstractHandler handler;
+        System.out.println("Enter document type:");
+        String type = scan.next();
+        switch (type){
```

Comment:
Let's make it so, that you have to enter the full name of the document: "document.txt"
and define its format by name before passing in switch-case